packages/sdk/src/client/client.ts

```diff
@@ -584,6 +585,59 @@ export class Client {
     return this.conditions[condition];
   }
 
+  /**
+   * `broadcast` broadcasts the given payload to the given topic.
+   */
+  public broadcast<T, P extends Indexable>(
```

Comment:
I think we should use an interface like `doc.broadcast(topic, payload)` rather than `client.broadcast(doc, topic, payload)`, as it aligns with how a client subscribes to a broadcast event using `doc.subscribe(topic, handler)`. Without removing or changing the existing function, it seems possible to add another broadcast function in `document.ts` that calls this function.

Reply:
@chacha912 
Thank you for pointing this out!  However, I haven't yet figured out how to call a method in `Client` directly from the document. Is there any way to get an instance of `Client` by `actorId`? 

```javascript
// document.ts
  public broadcast(topic: string, payload: any): void {

    const actorId = this.changeID.getActorID()
    
    // have to get `Client` instance by actorId  in order to call method `broadcast`
    const client =  getClientByActorId(actorId) // may be something like this
    
```

Reply:
I think one possible approach is to add some attribute members (e.g., `client`) to the `Document` class for this. However, I'm not sure if this is the best idea 😅.

Reply:
@hackerwins @chacha912 
Based on what you've mentioned, when attaching a document using the client's `attach` method, we could probably set client reference, within the `Document` class. However, this also needs to be discussed as well.

Reply:
I agree that the `doc.broadcast()` interface seems more appropriate. This has led to the addition of a `client` property within the `Document` class. At present, I can't think of a better alternative approach for now. (cc. @hackerwins )